dissect/target/plugins/os/windows/tasks.py

```diff
@@ -507,3 +1010,97 @@ def parse_task(self, entry):
         for entry in task.get_triggers():
             grouped = GroupedRecord("filesystem/windows/task/grouped", [record, entry])
             yield grouped
+
+    def parse_atjob(self, entry: TargetPath) -> Iterator[TargetRecordDescriptor]:
+        """
+        Parse a scheduled task from an at.exe job file.
+
+        Args:
+            entry: The path to the job file.
+
+        Yields:
+            TaskRecord or GroupedRecord: The parsed at.exe job file.
+        """
+        try:
+            at_task_data = atjob.ATJOB_DATA(entry.open())
+            at_task = AtTask(at_task_data)
+        except Exception as e:
+            raise InvalidTaskError(e)
+
+        at_task_data.author = at_task_data.author.decode("utf-16-le").rstrip("\x00")
```

Comment:
Overwriting cstruct values may lead to problems, I think it is best to create a separate `AtTask` class, similar to the `XMLTask` class and do the parsing during initialization. Same holds for the parsing now done in `parse_xml_task()`, that should be part of `XMLTask`'s init, while this function and `parse_xml_task()` should just construct the record (call them `get_*` or something).